README.md

```diff
@@ -90,5 +90,32 @@ public void Monitor()
 }
 ```
 
+**Admistrator rights**
+
+Some sensors require administrator privileges to access the data.
+
+Restart your IDE with admin privileges or add the following to your application to force admin rights on run
+
+#### **`app.manifest`**
```

Comment:
Let's link to MSDN on how to add app.manifest to avoid poluting the readme.

Reply:
would this be ok?